Features/GoogleAnalytics/script.js

```diff
@@ -0,0 +1,35 @@
+/* global wpData, Cookies */
+
+import $ from 'jquery'
+import 'file-loader?name=vendor/js-cookie.js!js-cookie/src/js.cookie'
+
+const $gaOptoutLinks = $('.globalAction-optoutGa')
+const data = wpData
+
+const alreadyOptedOut = getOptoutCookie()
+
+if (alreadyOptedOut) {
+  $gaOptoutLinks.remove()
+  window['ga-disable-' + data.gaId] = true
+} else {
+  $gaOptoutLinks.on('click', function (e) {
+    e.preventDefault()
+
+    const confirmOptout = window.confirm(data.confirm)
+
+    if (confirmOptout) {
+      window['ga-disable-' + data.gaId] = true
+      window.alert(data.success)
+      setOptoutCookie()
+    }
+  })
+}
```

Comment:
I think we should keep it seperated since the script.twig is really only for the Google Analytics script tag and doesn't really have own logic in there. Also the GoogleAnalytics.php passes down some anonymizeIP functionality into the script.twig view.